lib/valueflow.cpp

```diff
@@ -417,7 +417,6 @@ static std::vector<ValueType> getParentValueTypes(const Token* tok,
         if (ftok && argn >= 0) {
             if (ftok->function()) {
                 std::vector<ValueType> result;
-                std::vector<const Variable*> argsVars = getArgumentVars(ftok, argn);
```

Comment:
This was a bitch to reduce - super weird: https://trac.cppcheck.net/ticket/11218